src/subapps/projects/components/TableComponent.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+const TableComponent: React.FC<{}> = () => {
```

Comment:
is it used anywhere? maybe we don't need it after all

Reply:
I guess, no. Will remove.